cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -0,0 +1,79 @@
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
```

Comment:
Exactly -- the explicit quoting makes it a string in yaml.  That may be okay, but we should be consistent and eyes open here.